src/Files.App/Actions/Content/Run/RunAsAdminAction.cs

```diff
@@ -7,6 +7,8 @@ namespace Files.App.Actions
 {
 	internal sealed class RunAsAdminAction : BaseRunAsAction
 	{
+		private readonly IContentPageContext _context;
```

Comment:
Use this syntax and put this in the base abstract class to share among derived classes.

```suggestion
		private readonly IContentPageContext ContentPageContext = Ioc.Default.GetRequiredService<IContentPageContext>();
```